box-content-sdk/src/main/java/com/box/androidsdk/content/models/BoxUser.java

```diff
@@ -207,6 +202,7 @@ public String getAddress() {
      *
      * @return the URL of the user's avatar.
      */
+    @Deprecated //Use BoxApiUser.getDownloadAvatarRequest
```

Comment:
Can you add the deprecate info on the javadoc as well ? I believe this comment on this line will not be shown for developers on their IDE. They will probably only get a warning for this deprecate annotation.